.github/codeql/codeql-config.yml

```diff
@@ -7,4 +7,7 @@ paths:
 - '**/extension/src/**'
 
 paths-ignore:
-- '**/.vscode-test/**'
+  - '**/.vscode-test/**'
+  - '**/.vscode-test/**'
+  - '**/AutoLispExt/out/**/*.js'
```

Comment:
I guess you meant **/AutolispExt/out/test/**

Reply:
No, we should not scan the compiled js file under out folder.

Reply:
I see.